scripts/process/process.states.R

```diff
@@ -0,0 +1,26 @@
+
+#' clip/reduce the actual states that are used, add a data.frame to them that will be used by `visualize`
+#' 
+#' @param
+#' @param
+process.storm_states <- function(viz = as.viz('storm-states')){
+  sp <- readDepends(viz)[['states']]
+  add_svg_data(viz, sp)
+}
+
+add_svg_data <- function(viz, sp){
+  library(dplyr)
+  data.out <- data.frame(id = NA_character_, class = rep('state-polygon', length(sp)), raw.name = names(sp), stringsAsFactors = FALSE) %>% 
```

Comment:
assuming `state-polygon` is a generic term for the polygon and won't make a difference when this is used for islands?

Reply:
whoop, good catch.